Chummer/data/gear.xml

```diff
@@ -13769,6 +13839,66 @@
       <avail>8F</avail>
       <cost>200</cost>
     </gear>
+    
+   
+      <!-- Adds the drug grades as a plugin for drugs. -->
+    <gear>
+      <id>f22e79fa-fb04-4369-a761-a1a46e242bc8</id>
+      <name>Street Cooked</name>
+      <category>Drug Grades</category>
+      <rating>0</rating>
+      <source>CF</source>
+      <page>190</page>
+      <avail>0</avail>
+      <capacity>0</capacity>
+      <requireparent />
+      <cost>Parent Cost * -.5</cost>
```

Comment:
This is a bit dicey because it can create situations where someone can afford to buy a Street Cooked grade drug but not the Standard grade one. This is because the process of buying gear with some plugin after character creation is that you buy the base gear first, get the cost of the base gear deducted from your nuyen, and then buy the plugin and have its cost deducted from your nuyen. A bit of an edge case scenario, but still worth noting.